oak_functions/loader/src/server.rs

```diff
@@ -88,97 +88,85 @@ impl Policy {
     }
 }
 
+/// Enum indicating a serialization error
+enum SerializationError {
+    ByteArrayTooLarge,
+    FailedEncoding(prost::EncodeError),
```

Comment:
I don't think this will actually ever happen. The error indicates that the buffer was too small, but using a mutable Vec as the buffer means that it will be resized as needed. https://docs.rs/prost/0.7.0/prost/struct.EncodeError.html

Reply:
Thanks. Removed this enum.